modules/venat-module-market/src/data/universalis.ts

```diff
@@ -0,0 +1,32 @@
+import axios, { AxiosResponse } from 'axios';
+import { LookupResult } from './result';
+
+export interface UniversalisMarketInfo {
```

Comment:
Should this be split into a cross-module library? Or something to do later as needed?

Reply:
Universalis seems much less likely to be needed as a cross-module dependency compared to XIVAPI (or whatever other source we use later). We can extract it, if we think it will be needed in that way.